en/identity-server/7.2.0/docs/apis/restapis/application.yaml

```diff
@@ -392,7 +391,18 @@ paths:
       description: >
         This API provides the capability to create an application based on the
         information provided in an XML, YAML, or JSON file. <br>
-          <b>Scope(Permission) required:</b> `internal_application_mgt_create`
+        
+        <b>Scope(Permission) required:</b> 
+        
+          - `internal_application_mgt_create`
+        
+        <b>➕ Additional Scopes</b>
+        
+          To create an application with an authentication script via file import, you also need the following additional scope:
+        
+          | Action | Scope |
+          |---------|--------|
+          | Create an application with an authentication script via file import | `internal_application_script_update` |
```

Comment:
Since we have a separate endpoint for script update, do we need to mention this here ?

Since the new recommendation is to use the new endpoint to update the authentication script, I think we can avoid mentioning operation scopes in this API.

Reply:
added the recommendation to use new endpoint https://github.com/wso2/docs-is/pull/5662/commits/01140c92cbcdbac5af6d115fdac0a39449128049